styles/components/footer.css

```
@@ -0,0 +1,13 @@
+footer {
+  background-color: #262a47;
+  color: aliceblue;
+  text-align: center;
+  font-size: 20px;
+  border-top-left-radius: 20px;
+  border-top-right-radius: 20px;
+}
+
+footer p {
+  padding-top: 20px;
+  margin: 0px;
+}
```

You could be a bit more specific here and use classes as well:
```suggestion
.footer {
  background-color: #262a47;
  color: aliceblue;
  text-align: center;
  font-size: 20px;
  border-top-left-radius: 20px;
  border-top-right-radius: 20px;
}

.footer__text {
  padding-top: 20px;
  margin: 0px;
}
```